### ISSUE TITLE
Github to Jira Integration is not working properly

### ISSUE THREAD

**Issue by Michael-Xie** [atlassian/github-for-jira#1059]
Current Issues:
- the Smart Commits are not updated to their associated Jira issues in a timely manner
- Workflow is not working

Here are the current steps I followed and the above issues observed along the way:
1. Install the [App](https://marketplace.atlassian.com/apps/1219592/github-for-jira?hosting=cloud&tab=overview)
2. Follow the [instructions on install and configuring the App](https://github.com/atlassian/github-for-jira)![github_jira_setup](https://user-images.githubusercontent.com/6239834/151857202-972b0ccf-b0ff-46cc-8ad7-7ac5808c0f39.png)
3. Go to Board and Select an Issue
4. Copy the command to create branch from Development
5. Make changes in branch and add Smart Commit syntax
    a. It seems like the comment command works when written as the title of the commit and/or the body of the commit
```
Add vanilla Jira Smart Commit plugin WP-134 #review #comment trying out vanilla smart commit where everything resides on the title
```
```
build(package): WP-134 add Jira Smart Commit plugin 

- this one supports semantic versioning from someone who branced off digitalroute

WP-134 #review #comment Trying this plugin out
```
6. Make a pull request and merge commits
7. When the syncing was done, I saw:
![Screenshot from 2022-01-31 14-23-09](https://user-images.githubusercontent.com/6239834/151858890-918c75d5-b8a9-4ace-a972-7953b9025a1f.png)
where it seems to be able to associate with the right issue and writes the comments, but the status changes were executed
8. Update my Github to include the email that is associated with my Jira account, and update git config to use that email `git config --global user.email <my-email-used-in-jira>`
9. Make a Smart Commit, create Pull Request and merge commits to branch 
```
chore: remove unused committizen jira smart commit plugins 

WP-134 #review
```
Update wasn't reflected in Jira Board until the repos are manually resynced. However, the status did not get updated and the comments made before were duplicated.

**Comment by thombergs** [comment]
Hey @Michael-Xie,

the first screenshot you posted is not from this app but seems to be from some other GitHub integration app. Might it be that you have installed a different app than the "GitHub for Jira" app (there are quite a few)? We can only help you with the "GitHub for Jira" app. 

Let us know if you have sorted it out, we're happy to help!

Cheers,
Tom

**Comment by Michael-Xie** [comment]
> Hey @Michael-Xie,
> 
> the first screenshot you posted is not from this app but seems to be from some other GitHub integration app. Might it be that you have installed a different app than the "GitHub for Jira" app (there are quite a few)? We can only help you with the "GitHub for Jira" app.
> 
> Let us know if you have sorted it out, we're happy to help!
> 
> Cheers, Tom

We have Github for Jira installed. At the same from the screenshot below, it looks like there is another app that is installed, Github-Jira Integration. This one is probably what my admin went through. Thanks for pointing out on the first screenshot from my first post.
![Screenshot from 2022-01-31 15-55-02](https://user-images.githubusercontent.com/6239834/151873033-cc5d3a82-f94b-4eb9-97c1-33cd5720e0a2.png)

What would the installation process look like when installing Github for Jira?

**Comment by mboudreau** [comment]
@Michael-Xie After installing Github for Jira, have you added your github org to it in the configuration page?

**Comment by Michael-Xie** [comment]
> @Michael-Xie After installing Github for Jira, have you added your github org to it in the configuration page?

Yep, I got my Jira admin to configure with our github org. And have also uninstalled the other App that was causing a bunch of issues mentioned in the original post.

After that was set, I was able to have the Smart Commit work for associating issues to branch/commits, setting workflow statuses and comments.

Thanks again for help from @mboudreau and @thombergs for quickly catching the misinstallations from the plethora of Jira and Github integration apps.
